### PR TITLE
Owp 4

### PR DESCRIPTION
**Pull request by aleknow**

### CODE REVIEW

**Review by kfa-onewelcome — DISMISSED** [review]
iOS part

**Review by aleknow — COMMENTED** [review]
Looks good. When we fix Dexguard in the SDK it should be possible to revert mocking changes in the unit tests

**Comment by swetha-gadiraju** [comment]
> iOS part

File: lib/errors/error_codes.dart [hunk 1]
```diff
@@ -90,6 +90,7 @@ class PlatformErrorCodes {
 
   /// Android only
   static const String invalidDatetime = "9031";
+  static const String statelessRegistrationError = "9034";
```
[the remark this answers is not in the document]
Comment (swetha-gadiraju) [reply lib/errors/error_codes.dart:93]: I have made this change locally but haven't committed yet. Will do along with unit test changes.

**Review by aleknow — COMMENTED** [review]
looks good on Android side 👍

**Review by kfa-onewelcome — APPROVED** [review]
iOS part

**Pull request merged**